python/swe/dockerfiles/create_index.py

```diff
@@ -11,6 +11,8 @@
 from composio.utils.logging import WithLogger
 
 
+print("This is a Test PR")
```

Comment:
The placement of the print statement between imports and function definitions is unusual. Consider moving it to a more appropriate location within the file, such as inside a function or under a `if __name__ == '__main__':` block. This would improve code organization and readability.